modules/aioseop_sitemap.php

```diff
@@ -3796,7 +4202,18 @@ function get_all_post_type_data( $args ) {
 			}
 			$this->excludes = array_merge( $args['exclude'], $exclude_slugs ); // Add excluded slugs and IDs to class var.
 
-			// TODO: consider using WP_Query instead of get_posts to improve efficiency.
+			/**
```

Comment:
Just a reference instead.

```
/**
 * Filter to update the argumens before searching for posts.
 *
 * @since ?
 *
 * @param array $args {
 *     Arguments to send to WP_Query.
 *
 *     @see Parameter $args for WP_Query.
 *     @link https://codex.wordpress.org/Class_Reference/WP_Query#Parameters
 * }
 */
```